src/main/java/com/filmsage/filmsage/controllers/JournalController.java

```diff
@@ -14,53 +14,42 @@
 
 @Controller
 public class JournalController {
+    private JournalRepository journalDao;
     private UserRepository userDao;
-    private ReviewRepository reviewDao;
 
-    public JournalController(UserRepository userDao, ReviewRepository reviewDao){
+    public JournalController(UserRepository userDao, JournalRepository journalDao){
         this.userDao = userDao;
-        this.reviewDao = reviewDao;
+        this.journalDao = journalDao;
     }
 
-    @GetMapping("/users")
-    public String showUsers(Model model) {
-        model.addAttribute("allusers", userDao.findAll());
-        return "users/index";
+    @GetMapping("/journals")
+    public String showJournals(Model model) {
+        model.addAttribute("alljournals", journalDao.findAll());
+        return "/journals";
```

Comment:
these return statements point to our "templates" directory. "/journals" is just pointing to our "templates/journals" directory and not an actual template. suggestion: rewrite this to return "journals/index" and create an index.html template inside the journals directory.